templates/account/signup_vendor.html

```diff
@@ -0,0 +1,26 @@
+{% extends 'account/auth_base.html' %}
+{% load i18n %}
+{% load crispy_forms_tags %}
+{% load static %}
+{% load account socialaccount %}
+{% load socialaccount %}
+
+{% block subtitle %}Register as a Vendor{% endblock %}
```

Comment:
## ❓ SUGGSTION!
Try to be more general, don't represent any other project.

Reply:
The sign-up vendor wasn't needed for the boilerplate and was deleted.
I will review the other templates too, for other minor bugs that may have been overseen during initial integration of all auth templates.

The apple-call back-url bug in root settings has also been fixed.

Reply:
Thanks @hesham-04, you PR is accepted.